call_object.py

```diff
@@ -14,14 +14,14 @@
 
 
 while players[count].score != 500:
-    if global_cont ==1:
+    if global_verify ==1:
         count = 0
-        global_cont = 0
+        global_verify = 0
     players[count].distribute_again()
     players[count].PLay_game1()
```

Comment:
methods and variable names should be in `snake_case` refs [here](https://www.python.org/dev/peps/pep-0008/#:~:text=Use%20the%20function%20naming%20rules,invoke%20Python%27s%20name%20mangling%20rules.)
example:
```python
players[count].play_game()
```